modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -342,6 +343,8 @@ TensorMechanicsApp::associateSyntax(Syntax & syntax, ActionFactory & action_fact
   syntax.registerActionSyntax("EmptyAction", "BCs/Pressure");
   syntax.registerActionSyntax("PressureAction", "BCs/Pressure/*");
 
+  syntax.registerActionSyntax("GeneralizedPlaneStrainAction", "GeneralizedPlaneStrain/*");
```

Comment:
We intended to allow sub blocks under GeneralizedPlaneStrainAction. In terms of what the action does, it's better not being registered under TensorMechanics block.